changelogs/unreleased/luajit-grammar-fuzzer.md

```diff
@@ -0,0 +1,3 @@
+## feature/test/fuzz
+
+* LuaJIT now can be fuzzed using grammar-based fuzzer (gh-4823).
```

Comment:
```suggestion
* LuaJIT can now be fuzzed using a grammar-based fuzzer (gh-4823).
```

Reply:
it's a backport

Reply:
Well... I also agree, that this is a backport of the patch and it should be taken intact. OTOH, this patch has been committed prior to doc team is summoned to each PR to reduce changelog fixups when release comes. What I definitely don't want, that's typos in the resulting changelog. @andreyaksenov, @veod32, can we omit your suggestions for now, or you're not going to fix any grammar while tagging the release?

Reply:
OK, we'll proceed with no changes then.